node/test/python/aws-lambda-sdk/integration.test.js

```diff
@@ -36,7 +36,8 @@ describe('Python: integration', function () {
       'success',
       {
         variants: new Map([
-          // TODO: Cover v3.6 and v3.7 once support for those is fixed
+          // TODO: Cover v3.7 once support for it is fixed
+          ['v3-8', { configuration: { Runtime: 'python3.8' } }],
```

Comment:
So support for 3.7 is still pending? (title of the PR says otherwise)

Reply:
Totally understand the confusion. Yes, 3.7 is still pending in terms of the internal extension. Code is compatible though, so if customer wanted to import and manually set up instrumentation they can do it.

Reply:
@selcukcihan so can you clarify specifically, why it's not assumed as ready for v3.7. You say that it'll work for a customer, so why we do not assume it as ready?

Reply:
Because Python v3.7 runtime does not support setting `AWS_LAMBDA_EXEC_WRAPPER`. So we need to find another mechanism to wrap the customer handler. More details are in the "out of scope" section in the PR description.

Reply:
Ok, thanks for pointing that out. I think that means that Python 3.7 can only be instrumented manually. We will discuss it internally, for now let's stick to v3.8 (unless we find a way to instrument it automatically)